src/applications/find-va-forms/helpers/index.js

```diff
@@ -28,10 +28,12 @@ export const normalizeFormsForTable = ({ data: forms }) =>
     );
```

Comment:
Lots of great use cases for the nullish coalescing operator here but I don't think our project supports that yet